### DISCUSSION TITLE
VRoid SDK0.1.0 プレビュー版：日本語表記の追加

### DISCUSSION THREAD

**Discussion by FujiSunflower** [pixiv/vroid-sdk-developers#30] [General]
言語設定に同期する日本語表記を予め設定して頂けると嬉しいです。
セキュリティの観点からVRoidHubControllerプレハブを自作したくないのが理由です。

**言語設定**
![image](https://user-images.githubusercontent.com/20571538/147875435-cecd0b77-d182-4d7d-aaa3-3970a1a68a42.png)

**日本語表記が欲しい箇所**
![image](https://user-images.githubusercontent.com/20571538/147875487-9b7ed594-585b-48ca-8bff-3b630b3ae09f.png)

### その他
VRoid SDK0.1.0 プレビュー版公開おめでとうございます！
認証の際のコピー＆ペーストの手間が無くなったのが好感触です。

**Comment by nyamadan** [comment]
@FujiSunflower 
[nyamadan](https://github.com/pixiv/vroid-sdk-developers/discussions/3#discussioncomment-1877844)です。フィードバックありがとうございます。

> 言語設定に同期する日本語表記を予め設定して頂けると嬉しいです。

改めて確認なのですが言語設定を切り替えたときに、タブのテキストが「Pickup」→「ピックアップ」のように日本語になっていてほしいということでしょうか？

こちらのタブの箇所を確認しましたところ翻訳対象となっておりませんでした。
![Tabs](https://user-images.githubusercontent.com/20571538/147875487-9b7ed594-585b-48ca-8bff-3b630b3ae09f.png)

言語を切り替えたとしても何も変化がないとユーザーは本当に言語を切り替えられたのか心配になってしまうと思うので、こちらを翻訳対象に加えるように検討します。
また、英語テキストについてもVRoid Hubの言葉に合わせたほうが良いと思いますので英語のテキストについても見直しをいたします。

> VRoid SDK0.1.0 プレビュー版公開おめでとうございます！
> 認証の際のコピー＆ペーストの手間が無くなったのが好感触です。

ありがとうございます、もし気になる点などございましたらこちらでお気軽に相談くださいませ！

**Reply by FujiSunflower** [reply]
Yours → あなた
Liked → スキ
Pickup → 特集
のように日本語になって欲しいということで合っています。
